Readme.md

```diff
@@ -220,6 +220,30 @@ The `method` is optional. The default value is `link`, while the possible values
 
 When using the `copy` method, the import will take longer.
 
+### Importing and export single collections
+
+_**note:**_ Since directories might have been created by the docker daemon instead of a user,
+root privileges might be necessary.
+
+#### Exporting
+Liquid has to be deployed in order to export collections. When deployed, run
+```
+./liquid exportcollection <collection_name>
+```
+The exported collection data is stored in `<collections_directory>/exported/<collection_name`
```

Comment:
I think we should use a folder outside of the collections directory for exporting. Maybe the user mounts that directory read-only. Or maybe they have a collection called `exported` - journalists investigating cross-border trade perhaps? :) So how about we create a new folder, `exports`, in the liquid volumes folder (e.g. `/opt/node/volumes/exports`, and  write stuff in there?

Also, missed a closing `>` here ;)

Reply:
Okay I can see why that's a better place, good example ;)